src/components/Picklist/helpers/__test__/isScrolPositionlAtMenuBottom.spec.js

```diff
@@ -2,19 +2,21 @@ import isScrolPositionlAtMenuBottom from '../isScrolPositionlAtMenuBottom';
 
 describe('isScrolPositionlAtMenuBottom', () => {
```

Comment:
this has a typo between position and at there is an l
so change this name also scroll is wit double ll